config/locales/models/budget.en.yml

```diff
@@ -34,3 +34,34 @@ en:
     budget:
       edit: Edit budget
       new: Create budget
+  activerecord:
```

Comment:
I've probably said this already but I'd like to say it again: I think having a file per resource like this is excellent. It is contextual and you can see how language about the same resource is used in different places without distraction 👌